### PR TITLE
There is confusion between gigabytes and gigibytes. Let us standardize throughout.

### PR DESCRIPTION
**Pull request by lemire**
This explains the apparent discrepancy between our benchmark numbers.

### CODE REVIEW

**Comment by jkeiser** [comment]
How about we just report one, but make sure it's the same one across both? I grok the difference now, but I think output with both is just asking for comparison errors picking up the wrong one.

Probably best to just report GB rather than GiB, given that parse already reports that and we're already comparing version-to-version.

Along these same lines, I have been thinking we should simplify the output of the parse benchmark as well (at least by default). Give people one or two key numbers quickly and give more information with --verbose or --detailed.

**Comment by lemire** [comment]
> Probably best to just report GB rather than GiB, given that parse already reports that and we're already comparing version-to-version.

Ok I will edit the PR.

**Comment by lemire** [comment]
> Along these same lines, I have been thinking we should simplify the output of the parse benchmark as well (at least by default). Give people one or two key numbers quickly and give more information with --verbose or --detailed.

Better handled as a distinct PR/issue, but I agree.

Also, we should output the results using JSON. Duh.

**Comment by lemire** [comment]
@jkeiser  I revised all our benchmarks and documentation. We moved back and forth between different conventions regarding units. We should be consistent now.

I also took a stab at improving the docs while I was at it (just cleaning). 

I added a small section in the perf. file regarding Visual Studio. I wanted to tell folks that they should expect better performance with clang-cl. Please understand that it is not a way to dismiss cl.exe, which I think we should better support...

**Comment by lemire** [comment]
I have also been careful to leave traces of the fact that we refer to the official meanings of GB and GiB (with wikipedia as evidence) and not the "industry"... since the actual usage of the term GB varies... So it should be clear that we are deliberate in our choice and definition of units.

**Comment by lemire** [comment]
Failures are bogus. Waiting for @jkeiser's feedback before merging.

File: doc/performance.md [hunk 1]
```diff
@@ -140,3 +142,13 @@ few hundred megabytes per second if your JSON documents are densely packed with
 - When possible, you should favor integer values written without a decimal point, as it simpler and faster to parse decimal integer values.
 - When serializing numbers, you should not use more digits than necessary: 17 digits is all that is needed to exactly represent double-precision floating-point numbers. Using many more digits than necessary will make your files larger and slower to parse.
 - When benchmarking parsing speeds, always report whether your JSON documents are made mostly of floating-point numbers when it is the case, since number parsing can then dominate the parsing time.
+
+
+Visual Studio
+--------------
+
+On Intel and AMD Windows platforms, Microsoft Visual Studio enables programmers to build either 32-bit (x86) or 64-bit (x64) binaries. W urge you to always use 64-bit mode. Visual Studio 2019 should default on 64-bit builds when you have a 64-bit version of Windows, which we recommend.
```
Comment (gmile) [inline doc/performance.md:150]: Typo: "W urge" -> "We urge"

Comment (lemire) [reply doc/performance.md:150]: +1

**Comment by lemire** [comment]
> Serious note: if you have reason to prefer 1024, I'm fine with that too, I just suggested 1000 because I didn't want to have to mess around with perfdiff / checkperf :)

I don't care as long as we are consistent.

**Pull request merged**